doc/examples/ex52/ex52.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# GMT EXAMPLE 52
+#
+# Purpose:     Mixing images and overlaying on the Earth
+# GMT modules: grdgradient, grdimage, grdmath, grdmix, grdsample, solar
+
+gmt begin ex52 ps
+
+  # Use the location of the Sun at noon on Jun 22, 2000, Hawaii Standard Time
+  # Make a global 6m grid with a smooth transition across day/night boundary.
+  # We take advantage of the ATAN function to relatively quickly ramp from 0 to 1
+  gmt grdmath -Rd -I20m -r --PROJ_ELLIPSOID=sphere $(gmt solar -C -o0:1 -I+d2000-06-22T24:00+z-10) SDIST KM2DEG 90 SUB NEG 2 DIV ATAN DUP UPPER DIV 1 ADD 2 DIV = w.grd
```

Comment:
Define a variable `res=20m` and list the allowed values, so that users can easily try high-resolution data.